components/eventsrv-protocol/protocols/event.proto

```diff
@@ -65,12 +65,16 @@ message PackageIdent {
 message CensusEntry {
   /// Member-ID of originating Supervisor
   optional string member_id = 1;
+
   /// Service name
   optional string service = 2;
   /// Service group name
   optional string group = 3;
   /// Service group organization name
   optional string org = 4;
+  // NOTE: service group application and environment are further down
```

Comment:
Yup. I figured it might be easier for the next person that adds a field to this struct to be able to more easily see what the next available field number was by keeping the new ones down at the bottom.